src/design-system/components/Separator/Separator.tsx

```diff
@@ -6,27 +6,32 @@ import { useForegroundColor } from '../../color/useForegroundColor';
 export interface SeparatorProps {
   color: SeparatorColor;
   direction?: 'horizontal' | 'vertical';
+  thickness?: number;
 }
 
 /**
  * @description Renders a separator, either horizontal or vertical.
  */
-export function Separator({ color, direction = 'horizontal' }: SeparatorProps) {
+export function Separator({
+  color,
+  direction = 'horizontal',
+  thickness = 2,
```

Comment:
added ability to specify `Separator` thickness instead of strictly using `2`

Reply:
nice